### PR TITLE
feature:re-design-UI-for-user-authentication

### PR DESCRIPTION
**Pull request by Spectrumsun**
#### What does this PR do?
redesign the authentication page 

#### Description of Task to be completed?
redesign the authentication pages with material design

#### How should this be manually tested?
when the user clicks on the signup or signin page. They should see a redesigned page.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#156882161

#### Screenshots (if appropriate)
None

#### Questions:
None

### CODE REVIEW

**Review by emmabaye — COMMENTED** [review]
Good job with the redesign of the user authentication screen.

File: public/views/signup.html [hunk 1]
```diff
@@ -1,107 +1,90 @@
...
+        })
     })
-  })
-</script>
-
-
-
+  </script>
+  <script src='./../js/controllers/materialize.min.js' type="script"></script>
+
```
Comment (emmabaye) [inline public/views/signup.html:92]: There should be a new line at the end of this file.

File: public/views/signin.html [hunk 2]
```diff
@@ -1,70 +1,59 @@
...
+           $('#file').change(function(){
+             const fileName = $(this).val()
+             if(fileName){
+               $('.addimage').hide();
+             }
+           })
+         })
+     </script>
+     
+
```
Comment (emmabaye) [inline public/views/signin.html:59]: There should be a new line at the end of this file.

**Review by chuzksy-codeactive — COMMENTED** [review]
Nice work redesigning the signin and signup pages. I can see we are making progress here. Keep it up.

**Pull request merged**